evm_loader/lib/src/commands/collect_treasury.rs

```diff
@@ -34,22 +33,23 @@ pub fn execute(config: &Config, context: &Context) -> NeonResult<CollectTreasury
     info!("Main pool balance: {}", main_balance_address);
 
     let client = context
-        .rpc_client
-        .as_any()
-        .downcast_ref::<RpcClient>()
-        .expect("cast to solana_client::rpc_client::RpcClient error");
+        .blocking_rpc_client
+        .as_ref()
+        .expect("Non-blocking RPC client not initialized");
```

Comment:
I can't understand this part of code. Why we are using blocking RPC client in async? And the error message is also confusing.

Reply:
Because `solana_cli` requires a blocking client, sadly.

Reply:
In this case you need to fix only panic message to "blocking" instead of "non-blocking".